scripts/bulk-index.js

```diff
@@ -650,42 +710,51 @@ const run = async () => {
 
   // Enable direct-db access to Item, Bib, and Holdings services:
   overwriteModelPrefetch()
-
+  overwriteGeneralPrefetch()
+  overwriteSchema()
   // Require one of:
   // - csv
   // - bib/item id
   // - type, plus another qualifier (hasMarc or nyplSource)
-  if (argv.csv) {
-    await updateByCsv(argv)
-      .catch((e) => {
-        logger.error(`Error: ${e.message}`, e)
-      })
-  } else if (
-    argv.bibId ||
-    argv.itemId ||
-    (
-      argv.type &&
-      (
-        argv.hasMarc ||
-        argv.nyplSource ||
-        argv.fromDate
-      )
-    )
-  ) {
-    await db.initPools()
-    await updateByBibOrItemServiceQuery(argv)
-      .catch((e) => {
-        logger.error('Error ', e)
-        logger.error(e.stack)
-      })
-    db.endPools()
-  }
+  // if (argv.csv) {
+  //   await updateByCsv(argv)
+  //     .catch((e) => {
+  //       logger.error(`Error: ${e.message}`, e)
+  //     })
+  // } else if (
+  //   argv.bibId ||
+  //   argv.itemId ||
+  //   (
+  //     argv.type &&
+  //     (
+  //       argv.hasMarc ||
+  //       argv.nyplSource ||
+  //       argv.fromDate
+  //     )
+  //   )
+  // ) {
```

Comment:
Why disable all this?

Reply:
I didn't intend to leave it commented out, but in general I find a lot of the validation in this file hard to grok, and maybe unnecessary considering we often intent to run a reindex on everything